packages/core/package.json

```diff
@@ -126,6 +126,7 @@
     "cohere-ai": "^7.17.1",
     "date-fns": "^3.6.0",
     "dotenv": "^16.5.0",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
Add it as a dev dependency that way it gets bundled inline as it's only a couple of lines